docs/apb-filter-design.md

```diff
@@ -0,0 +1,80 @@
+# APB Filter Design Doc
+
```

Comment:
I'm conflicted because I think the spec should ultimately reflect the implementation as was committed. If we go with the PR and specifications directory approach, we can have a history of the conversation. Seeing how it began and how it ultimately ended.

I was planning on updating my auth spec to reflect the implementation. Then leave it be.